code/SoloSavingsApp/src/main/java/com/soloSavings/model/User.java

```diff
@@ -50,60 +49,9 @@ public class User {
     private Double balance_amount;
     private LocalDate last_updated;
 
-    public User(String username, String email, String password_plain_text) {
```

Comment:
IIRC this removes much of the work @hanwenzhang123  put together for the authentication checking.

Does this prevent auth?

Reply:
I see the logic being moved to the service layer `UserServiceImpl.java` and the rest being deleted was just setter and getters that we do not really need in the model, it looks good to me